src/main/java/lotto/presentation/controller/LottoController.java

```diff
@@ -0,0 +1,85 @@
+package lotto.presentation.controller;
+
+import lotto.application.dto.LottoTicketsDTO;
+import lotto.application.service.LottoService;
+import lotto.application.validation.BaseValidation;
+import lotto.application.validation.BonusNumberValidation;
+import lotto.domain.model.LottoNumbers;
+import lotto.domain.model.LottoTickets;
+import lotto.domain.result.LottoResult;
+import lotto.presentation.view.InputView;
+import lotto.presentation.view.OutputView;
+
+import java.util.List;
+
+
+public class LottoController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final BaseValidation<Integer> amountValidator;
+    private final LottoService lottoService;
+    private final BaseValidation<List<Integer>> lottoNumberValidator;
+    private final BonusNumberValidation bonusNumberValidator;
+
+    public LottoController(InputView inputView, OutputView outputView, BaseValidation<Integer> amountValidator, LottoService lottoService, BaseValidation<List<Integer>> lottoNumberValidator, BonusNumberValidation bonusNumberValidator) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.amountValidator = amountValidator;
+        this.lottoService = lottoService;
+        this.lottoNumberValidator = lottoNumberValidator;
+        this.bonusNumberValidator = bonusNumberValidator;
+    }
+
+    public void start() {
+        int amount = getValidAmount();
+
+        lottoService.purchaseLotto(amount);
+
+        LottoTickets lottoTickets = lottoService.getLottoTicketsFromRepository();
```

Comment:
메소드에 너무 많은 정보가 노출되고 있는 것 같습니다. 아래와 같이 바꾸어도 좋을 것 같아요
```suggestion
        LottoTickets lottoTickets = lottoService.getLottoTickets();
```

Reply:
좋은 지적 감사합니다!